libcore/FileUtils.vala

```diff
@@ -689,7 +700,7 @@ namespace Files.FileUtils {
                 break;
         }
 
-        return dt.format (format_string);
+        return string.join (" ", dt.format (format_string), " ");
```

Comment:
`get_formatted_date_time` already adds emspaces so I think we don't need to add spaces here?